apps/website/src/pages/docs/inputgroup.mdx

```diff
@@ -17,7 +17,7 @@ import * as AllExamples from 'examples';
   <AllExamples.InputGroupToggleSwitchExample client:load />
 </LiveExample>
 
-Input group class is a container to enhance an input by adding an icon, text, or a button in front or behind the input field as a "help text".
+Input group is a container to enhance an input by adding an icon, text, or a button in front or behind the input field as a "help text".
```

Comment:
i think input group is only for radio/checkbox/toggle rather than regular inputs, so it might be worth mentioning

Reply:
I was not prepared to update InputGroup docs because description is also wrong, just saw a small mistake in text. I can look into it more if you'd like. Or undo this change.